app/mailers/backorder_mailer.rb

```diff
@@ -3,9 +3,9 @@
 class BackorderMailer < ApplicationMailer
   include I18nHelper
 
-  def backorder_failed(order, linked_variants)
+  def backorder_failed(order)
     @order = order
-    @linked_variants = linked_variants
+    @linked_variants = order.variants
```

Comment:
This now lists all variants associated with an order, whether they're linked or not. Hmm, regardless we probably only want to know about variants that had errors. I'll wait to see where the PR goes..

Reply:
Yes, we don't actually know what users want. We have no data on that. In the pilots, I think, all variants will be linked. Looking at errors could be good but also another source for errors. I think that we should wait until this actually happens and what people say about it.